spec/rubocop/cop/style/redundant_condition_spec.rb

```diff
@@ -376,6 +376,34 @@ def test
           end
         RUBY
       end
+
+      it 'registers an offense and correct when the branches are the same with the same receivers' do
+        expect_offense(<<~RUBY)
+          result =
```

Comment:
That's just my two cents. Can you remove it? Because this assignment doesn't seem to matter.

```suggestion
```